CHANGELOG.md

```diff
@@ -35,7 +35,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - We added a minimal height for the entry editor so that it can no longer be hidden by accident. [#4279](https://github.com/JabRef/jabref/issues/4279)
 - We added a new keyboard shortcut so that the entry editor could be closed by <kbd>Ctrl<kbd> + <kbd>E<kbd>. [#4222] (https://github.com/JabRef/jabref/issues/4222)
 - We added an option in the preference dialog box, that allows user to pick the dark or light theme option. [#4130] (https://github.com/JabRef/jabref/issues/4130)
-
+- Allow the user to choose behavior after dragging and dropping files in Entry Editor using Preferences. [#4356]
```

Comment:
Please follow the convention of the other log entries

Reply:
Please let me know if this https://github.com/JabRef/jabref/pull/4408/commits/d57beda174ee0345aba61fb55e5f5963f3b51b9c looks good

Also, I have logged an issue to help.jabref.org to update the documentation for this new preference option - https://github.com/JabRef/help.jabref.org/issues/194